tests/python/pants_test/backend/python/tasks/test_python_repl_integration.py

```diff
@@ -21,3 +21,27 @@ def test_run_repl(self):
     pants_run = self.run_pants(command=command, stdin_data=program)
     output_lines = pants_run.stdout_data.rstrip().split('\n')
     self.assertIn('echo_interpreter_version loaded successfully.', output_lines)
+
+  @ensure_daemon
+  def test_run_repl_with_2(self):
+    # Run a Python 2 repl on a Python 2/3 library target.
+      command = ['repl',
+                'testprojects/src/python/interpreter_selection:echo_interpreter_version_lib',
+                '--python-setup-interpreter-constraints=CPython<3',
+                '--quiet']
+      program = 'from interpreter_selection.echo_interpreter_version import say_hello; say_hello()'
+      pants_run = self.run_pants(command=command, stdin_data=program)
+      version = pants_run.stdout_data.rstrip().split('\n')[0]
+      self.assertRegexpMatches(pants_run.stdout_data, r'2\.\d\.\d')
+
+  @ensure_daemon
+  def test_run_repl_with_3(self):
+    # Run a Python 3 repl on a Python 2/3 library target. Avoid some known-to-choke-on interpreters.
```

Comment:
indent this comment

Reply:
Actually it was the other lines the ones that were over-indented